index.js

```diff
@@ -0,0 +1,39 @@
+const express = require("express");
+const bodyParser = require("body-parser");
```

Comment:
express has built-in body parser. `app.use(express.json());` you can directly use that